demo_classification_tf.py

```diff
@@ -0,0 +1,69 @@
+import tensorflow as tf
+import numpy as np
+import cv2
+import argparse
+
+def sigmoid(x):
+  return 1 / (1 + np.exp(-x))
+
+def load_pb(path_to_pb):
+    with tf.io.gfile.GFile(path_to_pb, "rb") as f:
+        graph_def = tf.compat.v1.GraphDef()
+        graph_def.ParseFromString(f.read())
+    with tf.Graph().as_default() as graph:
+        tf.import_graph_def(graph_def, name='')
+        return graph
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--graph', help='.pb graph path', default = '../resnet_v2_101_299_frozen.pb')
+    parser.add_argument('--image', help='image path', default = '../realistic-blue-umbrella_1284-11412.jpg')
+    parser.add_argument('--labels', help='.txt labels path', default = '../classification_classes_ILSVRC2012.txt')
+    argv = parser.parse_args()
+
+    graph = load_pb(argv.graph)
+
+    labels = []
+    labels_filename = argv.labels
+
+    # Create a list of labels.
+    with open(argv.labels, 'rt') as f:
+        labels = f.read().strip().split('\n')  
+    
+    imageFile = argv.image
+    image = cv2.imread(imageFile)
+
+    image_mean = 127.5
+    image_std = 127.5
+    image_size_x = 299
+    image_size_y = 299 
+
+    image = cv2.resize(image, (image_size_x, image_size_y), interpolation = cv2.INTER_LINEAR)
+    img = image.reshape((1, 299, 299, 3))
+    img = (img - image_mean) / image_std
+
+    img = img.astype(np.float32)
+
+    output_layer = 'output:0'
+    input_node = 'input:0'
+
+    with tf.compat.v1.Session() as sess:
+        sess.graph.as_default()
+        tf.import_graph_def(graph.as_graph_def(), name = "")
+        prob_tensor = sess.graph.get_tensor_by_name(output_layer)
+        predictions, = sess.run(prob_tensor, {input_node: img})
+
+    predictions = sigmoid(predictions)
+
+    print_predictions = predictions
+    for i in range(5):
+        print()
+        print('Prediction ' + str(i) + ': ' + str(np.max(print_predictions)))
+        highest_probability_index = np.argmax(print_predictions)
+        print('Prediction index: ' + str(highest_probability_index))
+        print('Classified as: ' + labels[highest_probability_index-1])
+        print_predictions[highest_probability_index] = 0
```

Comment:
Let's do it more elegant. Take a look at [np.argsort](https://numpy.org/doc/stable/reference/generated/numpy.argsort.html?highlight=argsort#numpy.argsort)